EpikCord/__init__.py

```diff
@@ -3135,18 +3122,23 @@ async def send_modal(self, modal: Modal):
             f"/interactions/{self.id}/{self.token}/callback", json=payload
         )
 
+    @property
```

Comment:
Because you've made all of these properties, the command handler no longer works and errors, and that's because it's being called as a function, either get rid of this or change it

Reply:
letme update that ten